snapshot/snapshot.go

```diff
@@ -682,10 +712,12 @@ func (o *snapshotter) mounts(ctx context.Context, s storage.Snapshot, checkKey s
 
 }
 
+// Produces a file path like "{snapshotter.root}/snapshots/{id}/fs"
```

Comment:
nit: I think we should follow the normal godoc format and start with the name of the thing we're documenting. It makes it easier to export the function later.